kubernetes/k8_create.yml.erb

```diff
@@ -0,0 +1,60 @@
+# Deployment for create and redis service
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: create-deployment
+  labels:
+    app: create
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: create
+  template:
+      metadata:
+        labels:
+          app: create
+      spec:
+        containers:
+          - name: create-worker
+            image: "docker.ocf.berkeley.edu/create:<%= version %>"
+            imagePullPolicy: Always
+            ports:
+              - containerPort: 6378
+            resources:
+              limits:
+                cpu: 750m
+                memory: 512Mi
+            volumeMounts:
+              - mountPath: /home
+                name: nfs-export-home
+              - mountPath: /services
+                name: nfs-export-services
+              - mountPath: /etc/ocf-create
+                name: create-secrets
+                readOnly: true
+            livenessProbe:
+              exec:
+                command:
+                - python
+                - '-m'
+                - 'create.healthcheck'
+              initialDelaySeconds: 15
+              periodSeconds: 15
```

Comment:
maybe we should define an equivalent readiness probe

Reply:
From what I understand, a readiness probe would only signify when the pods are ready to receive traffic, but since celery only interacts through redis queues that wouldn't be necessary. So I left it out. If theres another reason for having it ill put it in

Reply:
when a new build is deployed, kube won't bring down the old containers until the new ones are ready

Reply:
Alright then I'll add it